Add more episode details to shownotes

tpikonen opened the pull request:
Add a line after subheading in shownotes with episode release date, length and size.

I usually have just the 'Released' column visible in the episode list TreeView, so having these visible in the shownotes is useful.

The formatting and the items in the details can be discussed in this PR and changed accordingly.

elelay commented:
Good idea! It seems maybe a bit crowded.

elelay commented:
Here is a screenshot on my desktop:
![capture](https://user-images.githubusercontent.com/222469/111035482-17032e80-841b-11eb-9b27-2bb8d39da002.png)

tpikonen commented:
The SailfishOS version of gPodder 4 has the episode details on the shownotes formatted like '2021-04-28 | 100.0 MiB | 35:00'.

I updated (and rebased) this PR to use this format and print the details with a smaller font. I'm going to use this on the adaptive branch, but it would be nice to have this everywhere.

elelay commented:
Looks good :+1:

![condensed](https://user-images.githubusercontent.com/222469/116457123-523ead00-a863-11eb-84f8-0bc1e9196e99.png)

auouymous commented:
LGTM, but 0.8 is too small to read. 0.9 is readable.

tpikonen commented:
I changed the scale to 0.9 and rebased.

auouymous commented:
`Today | 0.00 B | 00:00`

The columns use a hyphen for duration and nothing for size when they are unknown. Should this do something similar?

elelay commented:
> The columns use a hyphen for duration and nothing for size when they are unknown. Should this do something similar?

yes, it would be better

tpikonen commented:
The formatting is now the same as in columns. I'm somewhat itching to replace the "" for unknown size with a "-" though.

auouymous commented:
I agree with the "-", it looks better with or without a duration.

tpikonen commented:
Also added html.escape() to the HTML version.

elelay commented:
Merged, thanks :+1:

auouymous commented:
What about the unresolved if-else code review?
